snap/local/runtime-helpers/bin/source-env-file.sh

```diff
@@ -8,18 +8,17 @@ BINPATH="${ARGV[0]}"
 
 # binary name == service name/key
 SERVICE=$(basename "$BINPATH")
-if [ -z $ENV_FILE ]; then
-    ENV_FILE="$SNAP_DATA/config/$SERVICE/res/$SERVICE.env"
-fi
```

Comment:
This removes the possibility of passing $ENV_FILE from outside. The security-consun-bootstrapper and security-bootstrapper-redis apps use a different path, set inside the snapcraft.yaml file.